src/main/java/core/basesyntax/dao/impl/FruitTransactionDaoImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class FruitTransactionDaoImpl implements FruitTransactionDao {
+    @Override
+    public void addAll(List<FruitTransaction> fruitTransactions) {
+        Storage.fruitTransactions.addAll(fruitTransactions);
```

Comment:
you shouldn't store transactions in storage only fruits

Reply:
Ok